src/phoenix.js

```diff
@@ -0,0 +1,730 @@
+// Phoenix Channels JavaScript client
```

Comment:
@shosti yea I just realized that when I tried it. I tried including the github repo as an npm module which should work but they fucked up the format of the file so I can't import it :(